core/ast.h

```diff
@@ -432,21 +432,21 @@ struct Function : public AST {
     }
 };
 
-struct LiteralString;
-
 /** Represents import "file". */
 struct Import : public AST {
-    LiteralString *file;
-    Import(const LocationRange &lr, const Fodder &open_fodder, LiteralString *file)
+    // Must be a LiteralString.
```

Comment:
Because the desugarer visitor allows editing the AST *&, so everything has to be an AST *.